Formula/automysqlbackup.rb

```diff
@@ -56,4 +58,7 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
```

Comment:
Thanks for the quick response. A good test according to my understanding would run the command, check if databases are backed up, but would require the database credentials in the configuration file and hence cannot be implemented. 
So, I updated the test block to run `--help` instead.